### PR TITLE
fix(fix image): fix image

### LINKED ISSUE #2122: website:🐛 [Bug]: fix image in homepage
### Description

Image is not responsive
image

Image is having low quality in small sizes

[screencast-www.zesty.io-2023.11.15-18_12_00.webm](https://github.com/zesty-io/website/assets/44116036/0be578cf-fa34-43ea-b535-8d7867a8332d)

Grid feature lists of technologies not getting the response from api
image

### To Reproduce

```bash
1. Go to zesty.io or beta.zesty.io to look for unresponsiveness of the image
```

### Current vs. Expected behavior

_No response_

### Which area(s) are affected? (Select all that apply)

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by glespinosa**
fix image

# Description

1. Screenshot 1 fix: Try resizing the browser to look for the feature image becomes responsive
2. Screenshot 2 fix: Image is not blurring when browser is being resized
3. Screenshot 3 fix: It's now receiving properly the props from the api and the image is render correctly.

# Screenshots / Screen recording
Please add screenshots or recording if applicable

image

[screencast-test.zesty.io_3000-2023.11.15-18_57_36.webm](https://github.com/zesty-io/website/assets/44116036/5d6932d7-e1c7-4a41-ac5b-4ebe7e8a90db)

image

### CODE REVIEW

File: src/revamp/ui/FeatureBulletWithTestimonials/index.js [hunk 1]
```diff
@@ -141,15 +141,11 @@ const FeatureBulletWithTestimonials = ({
...
-            src={image}
-            loading="lazy"
-            alt={generateAlt('')}
-            width={788}
-            height={420}
-          />
-        </Box>
+          component="img"
+          src={image}
+          loading="lazy"
```
Comment (darwin808) [inline src/revamp/ui/FeatureBulletWithTestimonials/index.js:146]: use width and height

File: src/revamp/ui/HeroV2/index.js [hunk 2]
```diff
@@ -8,7 +8,7 @@ import { FONT_FAMILY } from 'components/globals/constants';
 import Logos from './Logos';
 
 const media =
```
Comment (darwin808) [inline src/revamp/ui/HeroV2/index.js]: revert this

Comment (glespinosa) [reply src/revamp/ui/HeroV2/index.js]: is it fine if it looks blurry when resizing?

File: src/revamp/ui/FeatureBulletWithTestimonials/index.js (hunk 1 above)
Comment (glespinosa) [reply src/revamp/ui/FeatureBulletWithTestimonials/index.js:146]: I added now default width and height , pls see below changes

**Pull request merged**